### PR TITLE
Doc: little fix

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -166,6 +166,7 @@ Very simple service: you give Navitia some coordinates, it answers you
 | url | Result |
 |----------------------------------------------|-------------------------------------|
 | `coords/{lon;lat}`                           | Detailed address point              |
+| `coverage/{lon;lat}/coords/{lon;lat}`        | Detailed address point              |
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: Can you add that it guesses the coverage from coord provided? (like what's done in other APIs)

**Comment by buildbotctp** [comment]
Build finished.

File: documentation/slate/source/includes/apis.md (hunk 1 above)
Comment (pbougue) [reply documentation/slate/source/includes/apis.md]: Like that one: https://github.com/CanalTP/navitia/blame/dev/documentation/slate/source/includes/apis.md#L1631

**Review by pbougue — APPROVED** [review]

**Comment by buildbotctp** [comment]
Build finished.

**Pull request merged**